src/melee.cpp

```diff
@@ -2273,6 +2251,39 @@ std::vector<special_attack> player::mutation_attacks(Creature &t) const
 
 std::string melee_message( const ma_technique &tec, player &p, const dealt_damage_instance &ddi )
 {
+    // Those could be extracted to a json
+
+    // Three last values are for low damage
+    static const std::array<std::string, 6> player_stab = {{
+        "You impale %s", "You gouge %s", "You run %s through",
```

Comment:
If it's just a variable being passed to _, it's probably a json string, which has been extracted elsewhere.
As long as the array is initialized inside a function it'll be fine, ```static type foo = something;``` has semantics of "do this the first time this code is executed", and if the statement is inside a function that means it happens after gettext is initialized, if it's *outside* a function is when it's a problem.